editor/components/post-featured-image/index.js

```diff
@@ -41,7 +44,10 @@ function PostFeaturedImage( { featuredImageId, onUpdateImage, onRemoveImage, med
 										naturalWidth={ media.media_details.width }
 										naturalHeight={ media.media_details.height }
 									>
-										<img src={ media.source_url } alt={ __( 'Featured image' ) } />
+										<img
+											alt={ __( 'Replace featured image.' + featuredImageAltText( media.alt_text ) ) }
```

Comment:
This won't work properly, you can't dynamically create a text that is passed to `__()` function. This probably should be rewritten to:

```js
alt={ __( 'Replace featured image.' ) + ' ' + featuredImageAltText( media.alt_text ) }
```